backend/functions/src/types.ts

```diff
@@ -0,0 +1,14 @@
+export type TaskTypeNoID = {
+  title: string;
+  description?: string;
+  createdAt?: string;
+  updatedAt?: Date;
+}
+
+export type TaskTypeWithID = {
+  title: string;
+  id: string;
+  description?: string;
+  createdAt?: string;
+  updatedAt?: Date;
+}
```

Comment:
```suggestion
export type TaskTypeWithID = TaskTypeNoID & { id: string;};
```

Reply:
do we need to declare the type `TaskTypeWithID` in the app?